Security/day03/Easy/chall-01-Lenna/LICENSE

```diff
@@ -0,0 +1,24 @@
+This is free and unencumbered software released into the public domain.
```

Comment:
What is this LICENSE ? why not one in root directory, and maybe you should use the MIT one

Reply:
This LICENCE comes from a pathwar template. It is used in every challenge related to pathwar